e2e/mock-api/src/app/routes.auth.js

```diff
@@ -440,7 +444,7 @@ export default function (app) {
   });
 
   app.get(authPaths.userInfo, wait, async (req, res) => {
-    if (req.path.includes('middleware')) {
+    if (req.headers['authorization'] && req.path.includes('middleware')) {
```

Comment:
We're not checking for the existence of the `authorization` header (`Bearer XXX`), but not checking its value since it is random.